Remotion/Web/Core/Utilities/HtmlUtility.cs

```diff
@@ -29,25 +28,28 @@ public static string Format (string htmlFormatString, params object[] nonHtmlPar
     {
       string[] htmlParameters = new string[nonHtmlParameters.Length];
       for (int i = 0; i < nonHtmlParameters.Length; ++i)
-        htmlParameters[i] = HtmlEncode (nonHtmlParameters[i].ToString());
+        htmlParameters[i] = WebString.CreateFromText (nonHtmlParameters[i].ToString() ?? string.Empty).ToString (WebStringEncoding.HtmlWithTransformedLineBreaks);
       return string.Format (htmlFormatString, (object[]) htmlParameters);
     }
 
+#if !NETFRAMEWORK
+    [Obsolete (
+        "HtmlUtility.Encode(string) is obsolete. Use WebString.ToString(WebStringEncoding.HtmlWithTransformedLineBreaks) instead. (Version 3.0.0)",
+        DiagnosticId = ObsoleteDiagnosticIDs.HtmlUtilityDiagID)]
+#endif
     public static string HtmlEncode (string nonHtmlString)
     {
-      string html = HttpUtility.HtmlEncode (nonHtmlString);
-      if (html != null)
-      {
-        html = html.Replace ("\r\n", "<br />");
-        html = html.Replace ("\n", "<br />");
-        html = html.Replace ("\r", "<br />");
-      }
-      return html;
+      return WebString.CreateFromText (nonHtmlString).ToString (WebStringEncoding.HtmlWithTransformedLineBreaks);
     }
 
+#if !NETFRAMEWORK
+    [Obsolete (
+        "HtmlUtility.Encode(string, HtmlTextWriter) is obsolete. Use WebString.Write(HtmlTextWriter) instead. (Version 3.0.0)",
+        DiagnosticId = ObsoleteDiagnosticIDs.HtmlUtilityDiagID)]
+#endif
     public static void HtmlEncode (string nonHtmlString, HtmlTextWriter writer)
     {
-      writer.Write (HtmlEncode (nonHtmlString));
+      WebString.CreateFromText (nonHtmlString).Write (writer);
     }
 
     private static readonly Regex s_stripHtmlTagsRegex = new Regex ("<.*?>", RegexOptions.Compiled);
```

Comment:
please make obsolete. there's no usage.

Reply:
The obsolete is missing for stripHtmlTags

Reply:
And what should I do with our 12 usages in remotion?

Reply:
ah, crap. missed that one. okay, we leave thsi for now